services/ui_backend_service/requirements.txt

```diff
@@ -1,7 +1,6 @@
-aiohttp==3.6.2
-aiohttp-swagger==1.0.15
-boto3==1.15.10
-pyee==8.0.1
-yarl==1.5.1
```

Comment:
is the latest yarl version functional again regarding url parameters?

Reply:
`aiohttp` is now working as expected without pinning down `yarl` version.